src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -422,7 +423,14 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             ScrollBar().Visibility(Visibility::Visible);
         }
 
-        _interactivity->UpdateSettings();
+        _interactivity.UpdateSettings();
+        if (_automationPeer)
+        {
+            _automationPeer.SetControlPadding(Core::Padding{ newMargin.Left,

```

Comment:
It doesn't seem like we're calling this when we create the automation peer -- we are not guaranteed to always call UpdateSettings _after_ creating a peer, so the first peer may have no padding.

Reply:
lemme toss this in the interactivity follow up bugs thread